test/integration/gobierto_exports/exports_page_test.rb

```diff
@@ -30,6 +30,21 @@ def test_index
       end
     end
 
+    def test_index_with_javascript
+      with_javascript do
+        with_current_site(site) do
+          with_stubbed_s3_file_upload do
```

Comment:
Why do we need `with_stubbed_s3_file_upload`? As far as I see the tests in this file only perform read operations, maybe we can remove this block from all the tests in this file?